packages/terra-navigation-side-menu/src/_MenuItem.jsx

```diff
@@ -133,7 +133,6 @@ class MenuItem extends React.Component {
           onKeyDown={this.handleKeyDown}
           onKeyUp={this.handleKeyUp}
           onBlur={this.handleOnBlur}
-          aria-haspopup={hasChevron}
```

Comment:
The fix here would likely not necessarily be to remove aria-haspopup. Only end points should use the 'link' role. Combined with this, it would make more sense to convert the list to use menu/menuitem roles, and the drill-in item use aria-haspopup.

Reply:
Might do some screen reader testing to see if the elements should be switched to `<a>` tags as well, or if divs are still fine.

Reply:
A combination of menu and menu-item roles works fine. As for the `<div>` and `<a>` tags , no difference in VO.